.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+`````release_notes
```

Comment:
This addition seems fine, @frikky let us know if other requirements for PR info should be included.